### ISSUE TITLE
UI and gophishing server on different servers

### ISSUE THREAD

**Issue by umairkhan10** [gophish/gophish#1461]
M working on a project and I want to use UI of gophishing on one of my server while my gophishing server running on another server. Can anyone help ?

**Comment by razermouse** [comment]
My understanding is that this is not possible out of the box. The web server that serves the admin pages to configure campaigns, landing pages, emails, and the connection to the SMTP server(s) is the same one that also serves the listening landing pages for active campaigns.

**Comment by riab-tam** [comment]
This should be doable by using an external MySQL database, installing gophish on your different servers, configuring the database connection settings to the same external MySQL database, and running one or the other (admin or phishing services) on each server.

**Comment by razermouse** [comment]
I actually got this working after getting my MySQL instance issue resolved. Since we use AWS EC2, I created an AMI image of my GoPhish application server and fired it up. Essentially, I'm running two servers with GoPhish on them. As I mentioned above, the GoPhish binary (or if you compiled from source) runs everything together but this does work. We use multiple domains so I pointed my Phishing domains at one GoPhish server and one of my other domains at the other one to act as the "console".   The last step is to configure the firewall to allow connections from only our networks to the console and disallow connections to 3333 on the landing page server.
